cli/src/cli/package.rs

```diff
@@ -332,14 +383,11 @@ fn unpack_recurse(mut obj: Object, to: &PathBuf) -> DefaultResult<()> {
 
 #[cfg(test)]
 // too slow!
-#[cfg(feature = "broken-tests")]
 mod tests {
     use super::*;
-    use crate::cli::init::tests::gen_dir;
-    use assert_cmd::prelude::*;
-    use std::{path::PathBuf, process::Command};
 
     #[test]
+    #[cfg(feature = "broken-tests")]
```

Comment:
Why is this test broken now?

Reply:
The whole package test suite has been broken for a while. In order to add another test I had to remove the root "broken tests" feature, add a new test but then add this feature to every other test

Reply:
ok, got it, that makes sense.